apps/wallet-mobile/src/yoroi-wallets/utils/utils.test.ts

```diff
@@ -84,6 +84,48 @@ describe('Quantities', () => {
     expect(Quantities.max('3', '2', '1')).toEqual('3')
     expect(Quantities.max('1', '1')).toEqual('1')
   })
+  it('fromInput', () => {
+    const english = {
+      prefix: '',
+      decimalSeparator: '.',
+      groupSeparator: ',',
+      groupSize: 3,
+      secondaryGroupSize: 0,
+      fractionGroupSize: 0,
+      fractionGroupSeparator: ' ',
+      suffix: '',
+    }
+
+    const italian = {
+      ...english,
+      decimalSeparator: ',',
+      groupSeparator: ' ',
+    }
+
+    BigNumber.config({
+      FORMAT: italian,
+    })
+
+    expect(Quantities.fromInput('', 3, italian)).toEqual(['0', '0'])
+    expect(Quantities.fromInput('1', 3, italian)).toEqual(['1', '1'])
+    expect(Quantities.fromInput('123,55', 3, italian)).toEqual(['123,55', '123.55'])
+    expect(Quantities.fromInput('1234,6666', 3, italian)).toEqual(['1 234,666', '1234.666'])
+    expect(Quantities.fromInput('55,', 3, italian)).toEqual(['55,', '55'])
+
+    expect(Quantities.fromInput('ab1.5c,6.5', 3, italian)).toEqual(['15,65', '15.65'])
+
+    BigNumber.config({
+      FORMAT: english,
+    })
+
+    expect(Quantities.fromInput('', 3, english)).toEqual(['0', '0'])
+    expect(Quantities.fromInput('1', 3, english)).toEqual(['1', '1'])
+    expect(Quantities.fromInput('123.55', 3, english)).toEqual(['123.55', '123.55'])
+    expect(Quantities.fromInput('1234.6666', 3, english)).toEqual(['1,234.666', '1234.666'])
+    expect(Quantities.fromInput('55.', 3, english)).toEqual(['55.', '55'])
+
+    expect(Quantities.fromInput('ab1.5c,6.5', 3, english)).toEqual(['1.56', '1.56'])
```

Comment:
`Quantity` is always `atomic` -> the expected output **must be** `['1.56', '1560']`, everything else is looking good.
```suggestion
    expect(Quantities.fromInput('ab1.5c,6.5', 3, english)).toEqual(['1.56', '1.56'])
```

Reply:
I'm also thinking that maybe this is not part of the `Quantities`, maybe we should have another helper `Denomination.` `fromText`, cuz quantities were created to don't care about the decimals itself.

Reply:
Now it returns an atomic Quantity.